main/src/ca/uwaterloo/flix/language/ast/TokenKind.scala

```diff
@@ -17,18 +17,323 @@ package ca.uwaterloo.flix.language.ast
+
+  case object KeywordBool extends TokenKind
+
+  case object KeywordCase extends TokenKind
+
+  case object KeywordCatch extends TokenKind
+
+  case object KeywordChan extends TokenKind
+
+  case object KeywordChar extends TokenKind
+
+  case object KeywordClass extends TokenKind
+
+  case object KeywordDef extends TokenKind
+
+  case object KeywordDeref extends TokenKind
+
+  case object KeywordDiscard extends TokenKind
+
+  case object KeywordElse extends TokenKind
+
+  case object KeywordEnum extends TokenKind
+
+  case object KeywordFalse extends TokenKind
+
+  case object KeywordFix extends TokenKind
+
+  case object KeywordFloat32 extends TokenKind
+
+  case object KeywordFloat64 extends TokenKind
+
+  case object KeywordForA extends TokenKind
+
+  case object KeywordForce extends TokenKind
+
+  case object KeywordForeach extends TokenKind
+
+  case object KeywordForM extends TokenKind
+
+  case object KeywordIf extends TokenKind
+
+  case object KeywordImport extends TokenKind
+
+  case object KeywordImpure extends TokenKind
+
+  case object KeywordInline extends TokenKind
+
+  case object KeywordInstance extends TokenKind
+
+  case object KeywordInt16 extends TokenKind
+
+  case object KeywordInt32 extends TokenKind
+
+  case object KeywordInt64 extends TokenKind
+
+  case object KeywordInt8 extends TokenKind
+
+  case object KeywordInto extends TokenKind
+
+  case object KeywordLat extends TokenKind
+
+  case object KeywordLaw extends TokenKind
+
+  case object KeywordLawful extends TokenKind
+
+  case object KeywordLazy extends TokenKind
+
+  case object KeywordLet extends TokenKind
+
+  case object KeywordMatch extends TokenKind
+
+  case object KeywordMod extends TokenKind
+
+  case object KeywordNamespace extends TokenKind
+
+  case object KeywordNil extends TokenKind
+
+  case object KeywordNot extends TokenKind
+
+  case object KeywordNull extends TokenKind
+
+  case object KeywordObject extends TokenKind
+
+  case object KeywordOpaque extends TokenKind
+
+  case object KeywordOr extends TokenKind
+
+  case object KeywordOverride extends TokenKind
+
+  case object KeywordPar extends TokenKind
+
+  case object KeywordPredicate extends TokenKind
+
+  case object KeywordPresent extends TokenKind
+
+  case object KeywordPub extends TokenKind
+
+  case object KeywordPure extends TokenKind
+
+  case object KeywordRead extends TokenKind
+
+  case object KeywordRecordRow extends TokenKind
+
+  case object KeywordRef extends TokenKind
+
+  case object KeywordRegion extends TokenKind
+
+  case object KeywordReify extends TokenKind
+
+  case object KeywordReifyBool extends TokenKind
+
+  case object KeywordReifyEff extends TokenKind
+
+  case object KeywordReifyType extends TokenKind
+
+  case object KeywordRel extends TokenKind
+
+  case object KeywordRem extends TokenKind
+
+  case object KeywordSchemaRow extends TokenKind
+
+  case object KeywordSealed extends TokenKind
+
+  case object KeywordSet extends TokenKind
```

Comment:
LiteralSet?

Reply:
We decided on using `SetHash` and so on for the rest of collection litterals